crowdstrike_identity_protect/lib/falconpy/_endpoint/_sample_uploads.py

```diff
@@ -0,0 +1,368 @@
+        "your customer account\n- `false`: File can be seen by other CrowdStrike customers \n\nDefault: `true`.",
+        "name": "is_confidential",
+        "in": "formData"
+      },
+      {
+        "type": "string",
+        "description": "A descriptive comment to identify the file for other users. "
+        "For example, use `--form comment=` when using cURL.",
+        "name": "comment",
+        "in": "formData"
+      }
+    ]
+  ],
+  [
+    "ExtractionListV1",
+    "GET",
+    "/archives/entities/extraction-files/v1",
+    "Retrieves the files extractions in chunks. Status `done` means that all files were processed successfully. "
+    "Status `error` means that at least one of the file could not be processed.",
+    "sample_uploads",
+    [
+      {
+        "type": "string",
+        "description": "The extraction operation ID.",
+        "name": "id",
+        "in": "query",
+        "required": True
+      },
+      {
+        "type": "integer",
+        "default": 0,
+        "description": "Max number of file extractions to retrieve.",
+        "name": "limit",
+        "in": "query"
+      },
+      {
+        "type": "string",
+        "description": "Offset from where to get file extractions.",
+        "name": "offset",
+        "in": "query"
+      }
+    ]
+  ],
+  [
+    "ExtractionGetV1",
+    "GET",
+    "/archives/entities/extractions/v1",
+    "Retrieves the files extraction operation statuses. Status `done` means that all files were processed successfully. "
+    "Status `error` means that at least one of the file could not be processed.",
+    "sample_uploads",
+    [
+      {
+        "type": "string",
+        "description": "The extraction operation ID.",
+        "name": "id",
+        "in": "query",
+        "required": True
+      },
+      {
+        "type": "boolean",
+        "default": False,
+        "description": "If `true` includes processed archive files in response.",
+        "name": "include_files",
+        "in": "query"
+      }
+    ]
+  ],
+  [
+    "ExtractionCreateV1",
+    "POST",
+    "/archives/entities/extractions/v1",
+    "Extracts files from an uploaded archive and copies them to internal storage making it available for content analysis.",
+    "sample_uploads",
+    [
+      {
+        "name": "body",
+        "in": "body",
+        "required": True
+      }
+    ]
+  ],
+  [
+    "GetSampleV3",
+    "GET",
+    "/samples/entities/samples/v3",
+    "Retrieves the file associated with the given ID (SHA256)",
+    "sample_uploads",
+    [
+      {
+        "type": "string",
+        "description": "The file SHA256.",
+        "name": "ids",
+        "in": "query",
+        "required": True
+      },
+      {
+        "type": "boolean",
+        "default": False,
+        "description": "Flag whether the sample should be zipped and password protected with pass='infected'",
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- 8e8c96d5-e189-4973-87ec-c079f0da7ffc -->

Reply:
This is a sample upload file of the falconpy SDK library.